controls/ssh_spec.rb

```diff
@@ -22,16 +22,19 @@
   command('ssh').exist?
 end
 
+ssh_custom_user = attribute('custom_user', value: 'root', description: 'The SSH user is not always root. It must be an unprivileged user in a container')
+ssh_custom_path = attribute('custom_path', value: '/etc/ssh', description: 'Sometimes ssh configuration files are present in another location and ssh use them with the -f flag')
+
 control 'ssh-01' do
   impact 1.0
   title 'client: Check ssh_config owner, group and permissions.'
-  desc 'The ssh_config should owned by root, only be writable by owner and readable to all.'
+  desc 'The ssh_config should owned by root or a specified user, only be writable by owner and readable to all.'
 
-  describe file('/etc/ssh/ssh_config') do
+  describe file(ssh_custom_path + '/ssh_config') do
     it { should exist }
     it { should be_file }
-    it { should be_owned_by 'root' }
-    it { should be_grouped_into os.darwin? ? 'wheel' : 'root' }
+    it { should be_owned_by custom_user }
```

Comment:
I think that needs to be `ssh_custom_user`

Reply:
I normally fixed that in the last commit.